daemon/metrics.go

```diff
@@ -26,12 +26,13 @@ func init() {
 		containerActions.WithValues(a).Update(0)
 	}
 	networkActions = ns.NewLabeledTimer("network_actions", "The number of seconds it takes to process each network action", "action")
-	engineVersion = ns.NewLabeledGauge("engine", "The version and commit information for the engine process", metrics.Unit("info"),
+	engineInfo = ns.NewLabeledGauge("engine", "The information related to the engine and the OS it's running on", metrics.Unit("info"),
 		"version",
 		"commit",
 		"architecture",
-		"graph_driver", "kernel",
-		"os",
+		"graph_driver",
```

Comment:
sounds good. we are good even if the format changes! Thanks for the feedback @stevvooe! As @aaronlehmann mentioned, we will be able to converge the `IDs` once we have `swarm mode enabled by default`.